content/en/docs/devops-user-guide/how-to-use/pipeline-webhook.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Trigger a Pipeline via a Webhook"
+keywords: 'Kubernetes, DevOps, Jenkins, Pipeline, Webhook'
+description: 'Learn how to trigger a Jenkins pipeline via a webhook.'
+linkTitle: "Trigger a Pipeline via a Webhook"
+weight: 11293
+---
+
+If you create a Jenkinsfile-based pipeline from a remote code repository, you can configure a webhook in the remote repository so that the pipeline will be automatically triggered when changes are made to the remote repository.
+
+This tutorial demonstrates how to trigger a Jenkins pipeline via a webhook.
```

Comment:
I'd prefer do not directly USE `Jenkins` here. `Pipeline` is fine. Jenkins is just the backend of the CI engine.

Reply:
Thanks for your comment. Fixed in [this commit](https://github.com/kubesphere/website/pull/1842/commits/325df0d4fe5a9ed1e33d04a44a23ada2e667629b).